packages/react-native/src/evm/providers/thirdweb-provider.tsx

```diff
@@ -76,7 +74,6 @@ export const ThirdwebProvider = <
 
   return (
     <ThirdwebProviderCore
-      thirdwebApiKey={thirdwebApiKey}
```

Comment:
@iketw Why is this removed? shouldn't it be passed along to ThirdwebProviderCore ?

Reply:
it is being passed down as part of `{...restProps}`, the only reason we specified it here was because we were providing a default api key

Reply:
@iketw restProps does not include `thirdwebApiKey`

Reply:
what do you mean? Because we removed thirdwebApiKey from the destructure here:

```
{
  children,
  createWalletStorage = createAsyncLocalStorage,
  supportedWallets = DEFAULT_WALLETS,
  authConfig,
  theme,
  ...restProps
}: PropsWithChildren<ThirdwebProviderProps<TChains>>
```

now `restProps` has the rest of the props, you can try doing `restProps.thirdwebApiKey`